.travis.yml

```diff
@@ -23,12 +16,6 @@ install:
 
 script:
 - go env
-- misspell -error $go_packages
-- go vet $go_packages
-- if [[ "$TRAVIS_GO_VERSION" != "tip" ]] ; then golint $go_packages; fi
```

Comment:
Note: I've centralised this test here https://github.com/kata-containers/tests/pull/43.